clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -1374,7 +1379,8 @@ public void handle(OffsetCommitResponse commitResponse, RequestFuture<Void> futu
                         } else if (error == Errors.TOPIC_AUTHORIZATION_FAILED) {
                             unauthorizedTopics.add(tp.topic());
                         } else if (error == Errors.OFFSET_METADATA_TOO_LARGE
-                                || error == Errors.INVALID_COMMIT_OFFSET_SIZE) {
+                                || error == Errors.INVALID_COMMIT_OFFSET_SIZE
+                                || error == Errors.UNKNOWN_TOPIC_ID) {
```

Comment:
That is right, thanks for pointing out. The resolution of topic name has been added to the response handler. If the topic is not defined, or the response topic is invalid because it contains neither an id or name, or contains both, that topic is ignored. The offset commit invocation is however not failed.